src/components/AssetMatching/components/Asset/AssetExternalLink.tsx

```diff
@@ -1,29 +1,20 @@
 import React, { FC } from 'react';
-import Tippy from '@tippyjs/react';
 import Styles from './Asset.style';
 import Icon from '../../../Icon';
-import 'tippy.js/dist/tippy.css';
+import Tooltip from 'components/Tooltip';
 
 export interface ExternalLinkProps {
   url: string;
   tooltipContent: string;
 }
 
-// @TODO: Introduce a generic tooltip component based on the ACs of the story: https://orfium.atlassian.net/browse/DS-153
-
 const AssetExternalLink: FC<ExternalLinkProps> = ({ tooltipContent, url }) => {
   return (
-    <Tippy
-      content={tooltipContent}
-      arrow
-      placement={'right'}
-      allowHTML={false}
-      css={Styles.tooltip}
-    >
+    <Tooltip content={tooltipContent} placement={'right'}>
```

Comment:
before: 
![Screenshot 2021-06-17 at 10 20 29 AM](https://user-images.githubusercontent.com/50381321/122350298-c5a49700-cf55-11eb-830d-dcb8f4d92c03.png)
after:
![Screenshot 2021-06-17 at 10 20 39 AM](https://user-images.githubusercontent.com/50381321/122350320-ca694b00-cf55-11eb-8234-00ffaee6fd84.png)

You should also pass the `Styles.tooltip` to the new tooltip component.

Reply:
dont worry about this, its configurable via theme provider. I already talked with elisavet and i would reach to you for the config needed